provider/cldr/src/transform/mod.rs

```diff
@@ -37,6 +38,7 @@ pub fn get_all_cldr_keys() -> Vec<ResourceKey> {
     result.extend(&numbers::ALL_KEYS);
     result.extend(&plurals::ALL_KEYS);
     result.extend(&time_zones::ALL_KEYS);
+    result.extend(&list::ALL_KEYS);
```

Comment:
Issue: Run `cargo make testdata` and it should generate the JSON files for `list@1/fr.json`.  You may need to add additional boilerplate in the other places in this file.